models/git/branch_test.go

```diff
@@ -21,7 +21,9 @@ func TestAddDeletedBranch(t *testing.T) {
 	assert.NoError(t, unittest.PrepareTestDatabase())
 	repo := unittest.AssertExistsAndLoadBean(t, &repo_model.Repository{ID: 1})
 	firstBranch := unittest.AssertExistsAndLoadBean(t, &git_model.Branch{ID: 1})
+	hash, err := git.ObjectFormatFromString(repo.HashType)
```

Comment:
```suggestion
	objectFormat, err := git.ObjectFormatFromString(repo.GitObjectFormat)
```

?

Reply:
Yes, I didn't yet finish the great rename :-) I will look carefully this morning where we still have `Hash` names